src/routes/__error.svelte

```diff
@@ -8,21 +8,33 @@
 </svelte:head>
 
 <section>
-  <img src="/assets/png/404-h.png" class="wide" alt="404 - Page Not Found" />
-  <img src="/assets/png/404-v.png" class="mobile" alt="404 - Page Not Found" />
+  <em>404</em>
+  <h1>Frank can't find where you're going!</h1>
+  <img src="/assets/png/lost-frank.png" class="wide" alt="404 - Page Not Found" />
+  <img src="/assets/png/lost-frank.png" class="mobile" alt="404 - Page Not Found" />
 </section>
 
 <style>
   section {
     min-height: 100vh;
     display: flex;
+    flex-direction: column;
     justify-content: center;
     align-items: center;
     margin: 0 24px;
+    color: var(--acm-light);
   }
-
-  section .mobile {
-    display: none;
+  section h1 {
+    margin-bottom: 2em;
+    align-items: center;
+    text-align: center;
+  }
+  section em {
+    align-items: center;
+    text-align: center;
+    font-size: 50px;
+    margin-bottom: 0.5em;
+    margin-top: 2em;
```

Comment:
maybe add one to the bottom too and i think we're gucci

Reply:
add more to*

Reply:
> add more to*

So you want me to increase the margin-bottom? Like 1 or 2em?

Reply:
ye the picture lookin kinda cramped on the bottom too if u make the window small enough

Reply:
<img width="1174" alt="image" src="https://user-images.githubusercontent.com/8981287/168750823-ae79b211-1693-4dac-994d-21d5a58c3d6e.png">

Reply:
![image](https://user-images.githubusercontent.com/95111582/168756838-addd4c46-1529-47cd-a8f8-d5b46160dc45.png)
How's this my boy

Reply:
2em too strong? Or 1 em?
![image](https://user-images.githubusercontent.com/95111582/168757039-8f36eebc-b843-4b64-8d0c-d369c0b48a8e.png)

Reply:
Disregard that first one, I didn't replicate the issue properly
![image](https://user-images.githubusercontent.com/95111582/168757566-a565e8d3-6e2b-4102-9520-bd23ec9fdf54.png)
, this is 3em

Reply:
3's probably fine

Reply:
Pushing now, I await your new bug that you will find soon 
![image](https://user-images.githubusercontent.com/95111582/168758903-8d74f435-3bd8-48fa-bdd3-00db7095b90f.png)